core/src/main/java/org/apache/brooklyn/core/mgmt/internal/AbstractManagementContext.java

```diff
@@ -372,6 +377,27 @@ public BrooklynProperties getBrooklynProperties() {
         return configMap;
     }
 
+    public StringConfigMap getConfigWithExternalConfigResolved() {
+        final ExternalConfigSupplierRegistry externalConfigProviderRegistry = getExternalConfigProviderRegistry();
+        if (externalConfigProviderRegistry == null)
+            return configMap; // no external config available yet, so return config without resolving
```

Comment:
This check is probably superfluous. It was required in an earlier iteration of this code where the constructor had possibly not finished, but that should not be the case now. I will either remove it, or (to be safe) make it throw an `IllegalStateException`.